src/main/java/com/avaya/letscode/CartOffer.java

```diff
@@ -0,0 +1,12 @@
+package com.avaya.letscode;
+
+public interface CartOffer extends Offer {
+    // For Cart level offers return discount applicable for the offer. 
+    // This function will be implemented in all derived classes for cart level offers
+    public default double getDiscountedValue(Cart cart) {
```

Comment:
"Public" is redundant in an interface as all methods of an interface are Public by default. So this can be removed from all the methods from all 3 interfaces.